lasagne/tests/layers/test_conv.py

```diff
@@ -622,6 +672,61 @@ def test_get_params(self, Conv3DImpl, DummyInputLayer):
         assert layer.get_params(_nonexistent_tag=False) == [layer.W, layer.b]
 
 
+class TestTransposedConv1DLayer:
+    @pytest.mark.parametrize(
+        "input, kernel, output, kwargs", list(transp_conv1d_test_sets()))
+    def test_defaults(self, DummyInputLayer, input, kernel, output, kwargs):
+        from lasagne.layers import TransposedConv1DLayer
```

Comment:
You'll need to wrap this in a `try..except`:
```python
try:
    from lasagne.layers import TransposedConv1DLayer
except ImportError:
    pytest.skip("TransposedConv1DLayer not available")
```
Same for the second test below. This ensures it will run through on Travis / on Theano 0.8.x.